composer.json

```diff
@@ -20,16 +20,16 @@
     },
     "autoload": {
         "psr-4": {
-            "EinsUndEins\\PluginTransactionMailExtender\\": "src/"
+            "EinsUndEins\\SmartInboxConnector\\": "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "EinsUndEins\\PluginTransactionMailExtender\\Tests\\": "tests/"
+            "EinsUndEins\\SmartInboxConnector\\Tests\\": "tests/"
         }
     },
     "extra": {
-        "shopware-plugin-class": "EinsUndEins\\PluginTransactionMailExtender\\TransactionMailExtender",
```

Comment:
@flagbird the namespace is not correct

Reply:
Good catch, even though it confused me for a minute by having a comment on the wrong side of the diff. :sweat_smile: 

Change is done.